go/oasis-node/cmd/debug/byzantine/tendermint.go

```diff
@@ -148,7 +74,7 @@ func (ht *honestTendermint) start(id *identity.Identity, dataDir string, useMock
 	}
 	ht.schedulerQuery = schedApp.QueryFactory().(*schedulerapp.QueryFactory)
 	// storage has no registration
-	roothashApp := roothashapp.New(timeSource, nil, 10*time.Second)
+	roothashApp := roothashapp.New(nil, 10*time.Second)
```

Comment:
Hm, the round timeout is not backed by consensus? I know this is not part of this PR but can you check and remove, I remember there being a `RoundTimeout` in the roothash genesis doc, but it seems it doesn't do anything?

Reply:
oh yeah it's wrong at the moment, some places use `RoundTimeout` from genesis, but the mux app still uses the passed in timeout :grin: will fix it

Reply:
fixed in separate pr: https://github.com/oasislabs/oasis-core/pull/2333 (it wasn't actually wrong, since in all cases (but here) the param form genesis was taken, just an unnecessary param)